packages/kbn-projects-solutions-groups/kibana.jsonc

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "shared-common",
+  "id": "@kbn/projects-solutions-groups",
+  "owner": "@elastic/kibana-core",
+  "devOnly": true
```

Comment:
This is included in production via `src/core/packages/root/server-internal/src/root/serverless_config.ts`

Reply:
Ouch, I started the package in `devOnly` mode, adding only types, but I ended up adding arrays too. I'll remove the `devOnly: true` and move it to platform/private.